GameData/ROCapsules/PartConfigs/Apollo D2/D2Parachute.cfg

```diff
@@ -0,0 +1,168 @@
+	}
+
+	scale = 1
+	rescaleFactor = 1.0
+
+	node_stack_bottom = 0.0, 0.0, 0.0, 0.0, -1.0, 0.0, 0
+	node_attach = 0.0, 0.0, 0.0, 0.0, -1.0, 0.0
+
+	attachRules = 1,0,1,0,0
+
+	//	============================================================================
+	//	Title, Description, Category, Techs
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = Utility
+	subcategory = 0
+
+	title = D-2 Parachute
+	manufacturer = Northrop Corp
+	description = Landing parachute system for the D-2 descent capsule.
+	tags = D2, Apollo, GE, Parachute
+
+	bulkheadProfiles = srf
+
+	sound_parachute_open = activate
+
+	//	============================================================================
+	//	Thermo, Durability
+	//	============================================================================
+
+	mass = 0.120
+	dragModelType = default
+	angularDrag = 0.1
+	crashTolerance = 8
+	breakingForce = 250
+	breakingTorque = 250
+	thermalMassModifier = 1.0
+	skinMassPerArea = 4
+	bodyLiftMultiplier = 0
+	stagingIcon = PARACHUTES
+	fuelCrossFeed = False
+	stageOffset = 1
+	childStageOffset = 1
+	buoyancyUseCubeNamed = PACKED
+
+	//	============================================================================
+	//	Animations and Textures
+	//	============================================================================
+
+	EFFECTS
+	{
+		rcpredeploy
+		{	AUDIO
+			{	channel = Ship
+				clip = sound_parachute_open
+				volume = 1
+			}
+		}
+		rcdeploy
+		{
+			AUDIO
+			{	channel = Ship
+				clip = sound_parachute_single
+				volume = 1
+			}
+		}
+		rccut
+		{	AUDIO
+			{	channel = Ship
+				clip = RealChute/Sounds/sound_parachute_cut
+				volume = 1
+			}
+		}
+		rcrepack
+		{	AUDIO
+			{	channel = Ship
+				clip = RealChute/Sounds/sound_parachute_repack
+				volume = 1
+			}
+		}
+	}
+
+	//	============================================================================
+	//	Modules and Resources
+	//	============================================================================
+
+	MODULE
+	{
+		name = RealChuteModule
+		caseMass = 0.08
+		timer = 0
+		mustGoDown = false
+		spareChutes = 2
+		cutSpeed = 0.5
+		invertCanopy = false
+		reverseOrientation = false
+
+		PARACHUTE
+		{
+			parachuteName = canopy
+			capName = cap
+			preDeploymentAnimation = semiDeployLarge
+			deploymentAnimation = fullyDeployLarge
+			material = Nylon
+			minIsPressure = False
+			minPressure = 0.01
+			preDeployedDiameter = 2
+			preDeploymentSpeed = 2
+			minDeployment = 3660
+			deployedDiameter = 50
+			deploymentSpeed = 6
+			deploymentAlt = 3350
+			cutAlt = -1
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleDragModifier
+		dragCubeName = SEMIDEPLOYED
+		dragModifier = 2
+	}
+
+	MODULE
+	{
+		name = ModuleDragModifier
+		dragCubeName = DEPLOYED
+		dragModifier = 4
+	}
+}
+
+@PART[ROC-D2Parachute]:AFTER[zzzRealismOverhaul]
+{
+	@maxTemp = 3600
+	@skinMaxTemp = 3600
```

Comment:
Parachute housing is made of tungsten?

Reply:
i think this was because the parachute housing was melting during reentry